app/frontend/libs/file-saver.js

```diff
@@ -0,0 +1,218 @@
+/* eslint-disable */
```

Comment:
Cannot we use not-transpiled version of the dependency, so that the result is more readable? 

https://github.com/eligrey/FileSaver.js/blob/master/src/FileSaver.js